rest-service/manager_rest/rest/resources_v2/nodes.py

```diff
@@ -80,8 +85,33 @@ def get(self, _include=None, filters=None, pagination=None,
                     scale_by *= group['properties']['planned_instances']
             node.set_actual_planned_node_instances(
                 scale_by * node.planned_number_of_instances)
+        if instance_counts:
+            if not filters.get('deployment_id'):
```

Comment:
Why does this need to have the dep id requirement? It looks like the nodes_list doesn't.

Reply:
just to limit it. I don't want to pull in too many node-instances. Like, this is a pretty expensive operation. In RD-4408 we can consider doing something for >1 deployments.

Reply:
<thumbs up>